examples/plot_draw_legend_docs.py

```diff
@@ -0,0 +1,107 @@
+"""
+Custom Legends with EarthPy
+===========================
+
+
+"""
+
+###############################################################################
+# Plotting with EarthPy
+# ---------------------
+#
+# .. note:: The examples below will show you how to make use of the draw_legend function for creating plots
+
+
+###############################################################################
+# Plot Continuous Data
+# ------------------------------
+#
+# Let's explore a simple plot using earthpy. To begin, import the needed packages
+# and create an array to be plotted. Below we plot the data as continuous with a colorbar
+# using the ``plot_bands()`` function
+
+
+import matplotlib.pyplot as plt
+import earthpy.plot as ep
+import numpy as np
+
+# Create a numpy array. Let's pretend this is what you want to plot.
+arr = np.random.randint(4, size=(5, 5))
+
+# When plot_bands is updated a cbar will be here as well
+ep.plot_bands(arr)
+plt.show()
+
+###############################################################################
+# Create Custom Discrete Legends with Earthpy
+# If you want to create a custom categorical legend, you can use the ``ep.draw_legend()`` function.
+
+f, ax = plt.subplots()
+im = ax.imshow(arr)
+ep.draw_legend(im)
+plt.show()
```

Comment:
This currently renders in a way that cuts off the legend:

![image](https://user-images.githubusercontent.com/2664564/55168119-ddef2800-5137-11e9-8d5e-6a1bf5d6003d.png)

Reply:
i actually haven't been able to figure out how to make this work properly. no matter what the figsize is it seems to cut off the legend.  any ideas?